File: Backup/10092017/source/backend/models/Sensor.php
<?php

namespace backend\models;

use Yii;

class Sensor extends \common\models\SensorBase {

    /**
     * @inheritdoc
     */
    public function attributeLabels() {
        return [
            'id' => Yii::t('backend', 'ID'),
            'module_id' => Yii::t('backend', 'Module'),
            'cam_bien_dan_thu' => Yii::t('backend', 'Solar panels temp'),
            'cam_bien_bon_solar' => Yii::t('backend', 'Bottom of Solar tank'),
            'cam_bien_muc_nuoc_bon_solar' => Yii::t('backend', 'Solar tank level'),
            'cam_bien_nhiet_do_bon_gia_nhiet' => Yii::t('backend', 'Heater tank temp'),
            'cam_bien_ap_suat_bon_gia_nhiet' => Yii::t('backend', 'Heater tank pressure'),
            'cam_bien_buc_xa_dan_thu' => Yii::t('backend', 'Lingh intensity'),
            'cam_bien_nhiet_dinh_bon_solar' => Yii::t('backend', 'Top of Solar tank temp'),
            'cam_bien_tran' => Yii::t('backend', 'Over tank'),
            'cam_bien_nhiet_do_duong_ong_2' => Yii::t('backend', 'Pipeline temp 2'),
            'du_phong' => Yii::t('backend', 'Environment Temp'),
            'created_at' => Yii::t('backend', 'Created At'),
            'cam_bien_ap_suat_duong_ong' => Yii::t('backend', 'Pipeline pressure'),
            'cam_bien_nhiet_do_duong_ong_1' => Yii::t('backend', 'Pipeline temp 1'),
        ];
    }

    public function getReport($from, $to, $moduleId) {
        $sensors = Sensor::find()->where([
                    '>=', 'created_at', $from
                ])->andWhere([
                    '<=', 'created_at', $to
                        //'<=', 'created_at', date('Y-m-d 23:59:59', strtotime($to))
                ])->andWhere([
                    'module_id' => intval($moduleId)
                ])->all();
        return $sensors;
    }

}

File: socket/source/src/main/java/proscom/domain/Sensor.java
// ----------------------------------------------------------------------------
// Copyright 2016, LAPTRINH.VN.
// All rights reserved
// ----------------------------------------------------------------------------
// Change History:
//  2016.10.12  datnh
//     - Initial release
// ----------------------------------------------------------------------------
package proscom.domain;

public class Sensor {
	
	public InformationElement ie;
	
	public String cam_bien_dan_thu;
	public String cam_bien_bon_solar;
	public String cam_bien_muc_nuoc_bon_solar;
	public String cam_bien_nhiet_do_bon_gia_nhiet;
	public String cam_bien_ap_suat_bon_gia_nhiet;
	
	public String cam_bien_buc_xa_dan_thu;
	public String cam_bien_nhiet_dinh_bon_solar;
	public String cam_bien_tran;
	public String cam_bien_ap_suat_duong_ong;
	public String cam_bien_nhiet_do_duong_ong_1;
	public String cam_bien_nhiet_do_duong_ong_2;
	
	@Override
	public String toString() {
		return "Sensor [ie=" + ie + ", cam_bien_dan_thu=" + cam_bien_dan_thu
				+ ", cam_bien_bon_solar=" + cam_bien_bon_solar
				+ ", cam_bien_muc_nuoc_bon_solar="
				+ cam_bien_muc_nuoc_bon_solar
				+ ", cam_bien_nhiet_do_bon_gia_nhiet="
				+ cam_bien_nhiet_do_bon_gia_nhiet
				+ ", cam_bien_ap_suat_bon_gia_nhiet="
				+ cam_bien_ap_suat_bon_gia_nhiet + ", cam_bien_buc_xa_dan_thu="
				+ cam_bien_buc_xa_dan_thu + ", cam_bien_nhiet_dinh_bon_solar="
				+ cam_bien_nhiet_dinh_bon_solar + ", cam_bien_tran="
				+ cam_bien_tran + ", cam_bien_ap_suat_duong_ong="
				+ cam_bien_ap_suat_duong_ong
				+ ", cam_bien_nhiet_do_duong_ong_1="
				+ cam_bien_nhiet_do_duong_ong_1
				+ ", cam_bien_nhiet_do_duong_ong_2="
				+ cam_bien_nhiet_do_duong_ong_2 + "]";
	}
	
}
